addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlApi.java

```diff
@@ -61,14 +69,56 @@ public String getPrefix() {
     public ApiResponse handleApiAction(String name, JSONObject params) throws ApiException {
         switch (name) {
             case ACTION_IMPORT_FILE:
-            case ACTION_IMPORT_URL:
-                LOGGER.debug("Nothing to see here (yet) :)");
-                break;
+                {
+                    URI endpointUri = UriFromStr(params.getString(PARAM_ENDPOINT), PARAM_ENDPOINT);
+                    File file = new File(params.getString(PARAM_FILE));
+                    if (!file.exists() || !file.canRead()) {
```

Comment:
Move the `canRead` to the following if? "does not exist" is misleading in that case.